provisioning/deploy.sh

```diff
@@ -22,7 +22,7 @@ sed -i "s/ATLAS_VERSION/$ATLAS_VERSION/" $BEET_TEMPLATE
 sed -i "s/BUILD_VERSION/$BUILD_VERSION/" $BEET_TEMPLATE
 
 # Update box Vagrantfile hash on dev builds.
-[ $ATLAS_BOX = "$BEET_BOX-dev" ] && sed -i "s/abef0f6a03cc23bb3da842cd1d12aa50/ef43117dab9c8e5d10a6b7d72d2e3fc4/" $PACKER_VAGRANTFILE
+[ $ATLAS_BOX = "beet/dev" ] && sed -i "s/abef0f6a03cc23bb3da842cd1d12aa50/ef43117dab9c8e5d10a6b7d72d2e3fc4/" $PACKER_VAGRANTFILE
```

Comment:
I forgot to save the change to the hashes. Please update to: `ef550c44b71ef09513fe24c4b564c8a9/c704fbc0a4343dcd6778896a150604b5`